### PR TITLE
Add Back to TOC link to all files of 0.14.x

### PR DESCRIPTION
**Pull request by vinayak42**
**NOTE**: Please read the CONTRIBUTING.md guidelines before submitting your patch,
and ensure you followed them all:

https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md

### Summary

This PR follows from #984. Here, I have added Back to TOC link to all sections in all markdown files of app/0.14.x

### Full changelog

* [Edited docs to ...]
* [Added cool new feature ...]
* ...

### Issues resolved

Fix #XXX

### Checklist:

- [x] [Commit message & atomicity](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [x] Documentation 
- [x] Spellchecked my updates
- [x] Ready to be merged

### CODE REVIEW

File: app/0.14.x/plugin-development/plugin-configuration.md [hunk 1]
```diff
@@ -76,6 +79,9 @@ return {
   end
 }
 ```
+[Back to TOC](#table-of-contents)
```
Comment (coopr) [inline app/0.14.x/plugin-development/plugin-configuration.md]: Fenced code blocks should have a blank line before and after - please check your various additions and add blank lines as necessary, thanks.

Comment (coopr) [reply app/0.14.x/plugin-development/plugin-configuration.md]: https://help.github.com/articles/creating-and-highlighting-code-blocks/#fenced-code-blocks

File: app/0.14.x/getting-started/configuring-a-service.md [hunk 2]
```diff
@@ -135,3 +140,5 @@ Go to [Enabling Plugins &rsaquo;][enabling-plugins]
 [enabling-plugins]: /{{page.kong_version}}/getting-started/enabling-plugins
 [proxy-port]: /{{page.kong_version}}/configuration/#nginx-section
 [mockbin]: https://mockbin.com/
+
+[Back to TOC](#table-of-contents)
```
Comment (coopr) [inline app/0.14.x/getting-started/configuring-a-service.md]: The trailing "Back to TOC" looks strange on the Getting Started pages (at least on all of them except the last one).

configuring_a_service_-_v0_14_x___kong_-_open-source_api_management_and_microservice_management

**Comment by vinayak42** [comment]
@coopr Anything else that needs to be done?

**Comment by coopr** [comment]
Thanks for your great contribution @vinayak42 !

**Pull request merged**